build/submodules.props

```diff
@@ -36,6 +36,5 @@
   </PropertyGroup>
```

Comment:
I assume since `TestsRequiredTheSharedRuntime` is still above that we didn't solve that problem in this PR?

Reply:
Otherwise should we consider this file for deletion?

Reply:
https://github.com/aspnet/AspNetCore/issues/4321 is still unresolved. I have one change in the works that addresses part of the problem, but it's a tricky enough challenge that I didn't tackle it completely.